src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/StateTests.cs

```diff
@@ -344,14 +372,11 @@ public static void PostDecrypt_Decrypt()
             }
         }
 
-        private static void AssertEncryptedContentEqual(byte[] expected, byte[] actual)
+        private static void AssertEncryptedContentEqual(byte[] expected, byte[] actual, bool runOnDesktop)
```

Comment:
`runOnDesktop` doesn't seem to be used.  Since this method is now down to two calls to SequenceEqual it could be inlined to just be `Assert.Equal(expected, actual)`...